### ISSUE TITLE
Accessibility: More Modular Speed Control

### ISSUE THREAD

**Issue by me-cfs** [jellyfin/jellyfin-expo#661]
**Describe the feature**
Hi, I’d like the option to be able to set playback speeds such as 0.4 for accessibility reasons. I cannot watch videos at normal speed due to disability and even 0.5 is often too fast.
